### ISSUE TITLE
Migrating to Rsbuild

### ISSUE THREAD

**Issue by L4Ph** [hellomuthu23/planning-poker#105]
I use this tool on a regular basis and it's very helpful.

The following pull request is in progress to migrate to Vite, but it seems to have been left unmerged.

Therefore, taking into account the migration costs, the lack of changes, and stability, I think it would be a good idea to migrate to Rsbuild.

If there are no problems, I would like to start this work. What do you think?

@hellomuthu23 

https://rsbuild.dev/guide/migration/cra
https://github.com/hellomuthu23/planning-poker/pull/65

**Comment by hellomuthu23** [comment]
Yes, Vite migration has been put on hold. Feel free to go for esbuild!

**Comment by L4Ph** [comment]
No, not esbuild.

This is a build tool called **Rsbuild**, which is based on Rspack.
Since it is highly compatible with Webpack (CRA), I will try it out for now.

https://rspack.dev/

**Comment by hellomuthu23** [comment]
ok, I am more inclined to use vita as it has a wider reach and will be good for open-source contribution.

**Comment by L4Ph** [comment]
It's been a while!!!
I'm working on this in my repository right now, but some of the tests aren't passing.
Can you help me?

**Comment by hellomuthu23** [comment]
Sure, will take a look!

**Comment by hellomuthu23** [comment]
All tests seems to be passing in your branch, maybe try cleaning up your node_modules and try again.

![Image](https://github.com/user-attachments/assets/5774e1c8-ad59-4b39-9201-4ccea1d45c9d)

**Comment by L4Ph** [comment]
I think you're looking at a different branch.
Anyway, the way the test targets are called has changed since it became ESM, so I fixed it accordingly.

Currently, all tests pass.

**Comment by L4Ph** [comment]
I've created a pull request.
Can you please review it?
